typescript/src/services/google-play.ts

```diff
@@ -37,7 +37,8 @@ export function getAccessToken(params: S3.Types.GetObjectRequest) : Promise<Acce
         })
 }
 
-export function buildGoogleUrl(subscriptionId: string, purchaseToken: string, packageName: string) {
+export function buildGoogleUrl(subscriptionId: string | undefined, purchaseToken: string, packageName: string) {
+    if(subscriptionId == undefined) throw Error("subscriptionId is undefined");
```

Comment:
Always use [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) in JS unless there's a specific reason to use [standard equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality). It's also [good practice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else#description) to use block statements when working with `if`.

```suggestion
    if (subscriptionId === undefined) { throw Error("subscriptionId is undefined") };
```

Regarding throwing an error here, that seems to be the pattern used elsewhere in this repo (e.g. the `getAccessToken` function further up this file does that). Unless @alexduf or @frankie297 see any problems doing that, LGTM.

Reply:
Thanks @JamieB-gu for your comment. I addressed the suggestions you made. Also changed the place to throw the error in fetchGoogleSubscription. This function is used in 3 places in the code and in all of them, the general exception is caught.